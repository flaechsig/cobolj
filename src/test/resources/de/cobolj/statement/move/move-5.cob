IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO-WORLD.

DATA DIVISION.
  WORKING-STORAGE SECTION.
  01 WS-GROUP1.
     02 WS-ITEM1 PIC X(2) VALUE 'AB'.
     02 WS-ITEM2 PIC X(2) VALUE 'CD'.
  01 WS-GROUP2.
     02 WS-ITEM3 PIC 9(2) value 2.
     
PROCEDURE DIVISION.
DISPLAY WS-GROUP1 " " WS-GROUP2.
MOVE WS-GROUP1 TO WS-GROUP2.
DISPLAY WS-GROUP1 " " WS-GROUP2.
STOP RUN.