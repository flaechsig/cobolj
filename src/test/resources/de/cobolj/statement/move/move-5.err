Group Move mit komischen Daten. FIXME: Soll-Verhalten ist nicht sauber definiert