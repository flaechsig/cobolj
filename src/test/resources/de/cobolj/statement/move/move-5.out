ABCD 02
ABCD 00
